README.md

```diff
@@ -70,10 +70,18 @@ The versioning convention of ECMAScript has been the subject of hot debates. We
 
 ## Gulp
 
-[Gulp](http://gulpjs.com/) is a task runner aiming at automating tedious and possibly complex tasks. The idea behind Gulp (and its peer [Grunt](#grunt)) is to define tasks that perform (usually file-based) actions. These tasks can then be run through the command line or as part of another build step. 
+[Gulp](http://gulpjs.com/) is a task runner aiming at automating tedious and possibly complex tasks. The idea behind Gulp (and its peer [Grunt](#grunt)) is to define tasks that perform (usually file-based) actions. These tasks can then be run through the command line or as part of another build step.
 
 Gulp also owes its success to its very large ecosystem of plugins, making it easy to perform everyday’s tasks without having to write much code.
 
+## JSLINT
+
+[JSLint](http://www.jslint.com/) is a static analysis “code quality” tool for JavaScript developed by Douglas Crockford, a well-known developer also responsible for JSON, JSMin, ADSafe and parts of YUI.
```

Comment:
I would skip the whole Douglas Crockford part, as it is mostly irrelevant for someone just wondering about JSLint. I would jump directly to the explanation of linting.